docs/src/freeenergy.md

```diff
@@ -0,0 +1,635 @@
+
+```julia
+# MBAR.jl
+
+mbar_gen = assemble_mbar_inputs(C, B, states; # Coordinates, boundaries and thermodynamic states
+                                target_state = target_state, # The target state, in our case the unbiased system
+                                energy_units = energy_units)
+
+u        = mbar_gen.u         # Reduced energy matrix, K states by N sampled conformations
+u_target = mbar_gen.u_target  # The reduced energy of the N samples evaluated by the target state hamiltonian
+N_counts = mbar_gen.N         # Number of sampled conformations
+win_of   = mbar_gen.win_of    # Indexing helper that tells which k state was used to generate each n sample
+shifts   = mbar_gen.shifts    # Numerical shifts, if used, for stability reasons when building the reduced energy matrix
+```
+
+This generates the necessary inputs to use the self-consistent iteration method to solve the MBAR equations. Of course, Molly provides the [`iterate_mbar`](@ref) method to do so:
+
+```julia
+# MBAR.jl
+F_k, logN = iterate_mbar(u, win_of, N_counts) # Returns the relative free energy of each k state and log.(N_counts), needed for downstream computations
+```
+
+With this, we can also produce a weight matrix and a vector of target weights, using the  [`mbar_weights`](@ref) method, that will let us reweight any arbitrary quantity from the sampled K thermodynamic states to any target state:
+
+```julia
+# MBAR.jl
+W_s, w_target = mbar_weights(u, u_target, F_k, logN, N_counts; check = true, shifts = shifts) # Returns the weights matrix and the target weights
+```
+
+And finally, we can estimate the PMF using the output of the previous step by calling the [`pmf_with_uncertainty`](@ref) method:
+
+```julia
+# MBAR.jl
+
+pmf = pmf_with_uncertainty(u, u_target, F_k, N_counts, logN, CV; shifts = shifts, kBT = kBT)
+
+centers   = pmf.centers         # The collective variable 
+PMF       = pmf.F               # PMF in kBT
+PMF_enr   = pmf.F_energy        # PMF in energy units
+sigma     = pmf.sigma_F         # Standard deviation in kBT
+sigma_enr = pmf.sigma_F_energy  # Standard deviation in energy units
+```
+
+But what about the short path? Well, we also provide an overload of the [`pmf_with_uncertainty`](@ref) method that allows to get the PMF in a single call by doing:
+
+```julia
+# MBAR.jl
+pmf = pmf_with_uncertainty(C,            # Coordinates
+                           B,            # Boundaries
+                           states,       # Themodynamic states 
+                           target_state, # Target state
+                           CV)           # Collective Variable
+```
+
+Now one can plot this into a graph. The code for that is left as an exercise to the reader, but the results should look like something similar to this!
+
+![PMF along the dipeptide torsion in kBT units](images/dihedral_pmf_kbt.png)
```

Comment:
Does the blue line represent uncertainty? If so it might be worth mentioning, and saying that we have converged (or whatever the term is here).